lib/travis/build/data.rb

```diff
@@ -137,6 +137,10 @@ def branch
         job[:branch] || ''
       end
 
+      def tag
```

Comment:
@BanzaiMan that is on the `data` model, if you look at the `clone_args`, the `branch` method called isn't `data.branch`. All I was trying to mention in my comment is that you can't just use `tag`, you would need to use `data.tag` or create a `tag` method.